gym/envs/tests/test_envs.py

```diff
@@ -12,7 +12,12 @@ def should_skip_env_spec_for_tests(spec):
     # We skip tests for envs that require dependencies or are otherwise
     # troublesome to run frequently
 
-    # Skip mujoco tests for pull request CI
+    # Skipping if pull request
+    is_pr = os.environ.get('TRAVIS_PULL_REQUEST') != 'false' and os.environ.get('TRAVIS_PULL_REQUEST') is not None
```

Comment:
The `skip_mujoco` line below this should handle the case of a PR build. It might be buggy though. Are you sure the issue isn't from the `test_monitor_envs` file? Can you paste me a relevant build URL?